docs/api-docs/apps/guide/apps-10-buttons.md

```diff
@@ -23,7 +23,7 @@ First, embed an install button like the one below, at any web location from whic
 
 ![Install Button](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-10-buttons-01.png "Install Button")
 
-Redirect anyone who presses your button to: `https://login.bigcommerce.com/app/{{CLIENT_ID}}/install`.
+Redirect anyone who presses your button to: `https://login.bigcommerce.com/app/install`.

```

Comment:
Are you sure this is correct?

Reply:
This is not correct. I emailed Nathan on Friday asking him about the exceptions to the rule. He just got back with me today and this url and requests stays as it is as well as the others on this page.